### DISCUSSION TITLE
能否在配置文件中配置vpn portal

### DISCUSSION THREAD

**Discussion by yuyu1025** [EasyTier/EasyTier#449] [Q&A]
我在使用wireguard时，配置服务端没有找到对应的配置方式，做了一些尝试，但是都不行希望能解答一下。
![image](https://github.com/user-attachments/assets/95711829-be71-4f6e-9b12-710b279087f3)
![image](https://github.com/user-attachments/assets/b67a1044-e0d8-4de0-af32-2f243887a1db)

**Comment by KKRainbow** [comment]
用命令行带着参数启动一下，会打印对应的配置文件

**Comment by yuyu1025** [comment]
我连接上了WireGuard，十分感谢。但同时我有一个问题，自带的客户端与这个WireGuard相比好在哪里？还有必要去使用自带的客户端吗？

**Comment by KKRainbow** [comment]
wg 没法打洞但是支持全平台；自带客户端没有 ios 只有 安卓
